webdriver/tests/bidi/script/evaluate/invalid.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import webdriver.bidi.error as error
+from ... import recursive_compare
```

Comment:
Imports for local modules should be separated from the third party ones.

```suggestion
import webdriver.bidi.error as error

from ... import recursive_compare
```

Reply:
done